pkg/releaselock/rpc.go

```diff
@@ -41,6 +41,8 @@ const (
 	notifyCloseChannelSize = 1
 )
 
+type FileStoreRPCClient struct{}
```

Comment:
I’d recommend adding `var _ LockService = FileStoreRPCClient{}` before this line. The empty struct made me confused at the beginning.

Reply:
LockService is only relevant in controller.go